packages/solid-repl/package.json

```diff
@@ -21,36 +21,36 @@
     "solid-js": ">=1.7.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.22.19",
-    "@babel/plugin-syntax-jsx": "^7.22.5",
-    "@babel/preset-typescript": "^7.22.15",
-    "@babel/standalone": "^7.22.19",
-    "@babel/types": "^7.22.19",
-    "@types/babel__standalone": "^7.1.5",
-    "@types/dedent": "^0.7.0",
-    "@types/fs-extra": "^11.0.2",
-    "@unocss/cli": "^0.55.7",
-    "@unocss/preset-wind": "^0.55.7",
-    "@unocss/reset": "^0.55.7",
-    "babel-preset-solid": "^1.7.7",
-    "esbuild": "^0.19.3",
-    "eslint-solid-standalone": "^0.13.0",
-    "fs-extra": "^11.1.1",
-    "jiti": "^1.20.0",
-    "monaco-editor": "^0.43.0",
-    "prettier": "^3.0.3",
-    "solid-js": "1.7.11",
-    "typescript": "^5.2.2",
-    "unocss": "^0.55.7"
+    "@babel/core": "^7.24.1",
+    "@babel/plugin-syntax-jsx": "^7.24.1",
+    "@babel/preset-typescript": "^7.24.1",
+    "@babel/standalone": "^7.24.2",
+    "@babel/types": "^7.24.0",
+    "@types/babel__standalone": "^7.1.7",
+    "@types/dedent": "^0.7.2",
+    "@types/fs-extra": "^11.0.4",
+    "@unocss/cli": "^0.58.6",
+    "@unocss/preset-wind": "^0.58.6",
+    "@unocss/reset": "^0.58.6",
+    "babel-preset-solid": "^1.8.16",
+    "esbuild": "^0.20.2",
+    "eslint-solid-standalone": "^0.13.1",
+    "fs-extra": "^11.2.0",
+    "jiti": "^1.21.0",
+    "monaco-editor": "^0.47.0",
+    "prettier": "^3.2.5",
+    "solid-js": "1.8.16",
```

Comment:
Now the output show the current solid-js syntax, which in some cases is quite different from v1.7.11